Multisig/UI/App/Unlock/FaceIDUnlockViewController.xib

```diff
@@ -45,7 +45,7 @@
                 </label>
             </subviews>
             <viewLayoutGuide key="safeArea" id="fnl-2z-Ty3"/>
-            <color key="backgroundColor" name="backgroundPrimary"/>
+            <color key="backgroundColor" name="backgroundUnlock"/>
```

Comment:
can't we just go with backgroundPrimary ?

Reply:
backgroundPrimary is not completely white in day mode and not completely black in night mode, so you will see the difference in color between toolbar and content area.

Reply:
Maybe it's because wrong constraints ?

Reply:
no, it's the color issue